Problem5/src/problem5/Problem5.java

```diff
@@ -10,7 +14,47 @@
      * @param args the command line arguments
      */
     public static void main(String[] args) {
-        // TODO code application logic here
+        Scanner scanner = new Scanner(System.in);
+        
+        //get input from user
+        System.out.print("Type a line of text: ");
+        String input = scanner.nextLine();
+        String output = "";
+        
+        StringTokenizer tokenizer = new StringTokenizer(input);
+        
+        while (tokenizer.hasMoreTokens()) {
+            String token = tokenizer.nextToken();
+            
+            //replace quotes
+            token = token.replace("'", "\"");
+            
+            //replace all worlds ending with or with words ending in our
+            // - (or) predicate matches any seqence of characters equaling 'or'
+            // - (?=\W) predicate looks ahead to check wether the next symbols are part of a 'word' (matches any non alphabetic characters)
+            token = token.replaceAll("(or)(?=\\W)", "our");
```

Comment:
This matching rule only works if there is punctuation after it (I.e. the word color by itself does not convert to colour)